src/components/App.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
+// import calcualate from '../logic/calculate';
 
```

Comment:
- This is one of the requirements for this milestone, I noticed that this was commented out probably because of linter rule of unused vars, so add this back then disable the rule just for this file, then once you are using this variable in the `App` component, you can enable the rule